app/src/main/java/com/example/lunark/repositories/ReviewReportRepository.java

```diff
@@ -0,0 +1,76 @@
+package com.example.lunark.repositories;
+
+import android.util.Log;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import com.example.lunark.models.ReviewReport;
+import com.example.lunark.services.ReviewReportService;
+import com.example.lunark.util.ClientUtils;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+public class ReviewReportRepository {
+    private static final String LOG_TAG = "ReviewReportRepository";
+    ReviewReportService reviewReportService;
+
+    @Inject
+    public ReviewReportRepository() {
+        this.reviewReportService = ClientUtils.reviewReportService;
+    }
+
+    @Inject
+    public void setRetrofit(Retrofit retrofit) {
+        this.reviewReportService = retrofit.create(ReviewReportService.class);
+    }
```

Comment:
This can be written more concisely.
```suggestion
    @Inject
    public ReviewReportRepository(Retrofit retrofit) {
        this.reviewReportService = retrofit.create(ReviewReportService.class);
    }
```